foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -91,168 +91,171 @@ This function does NOT put your new element into the DOM - it creates it in memo
 
 #### Altering elements
 
-When you have a reference to an element, you can use that reference to alter the element's own properties. This allows you to do many useful alterations, like adding/removing and altering attributes, changing classes, adding inline style information and more.
+When you have a reference to an element, you can use that reference to alter the element's own properties. This allows you to do many useful alterations, like adding, removing, or altering attributes, changing classes, adding inline style information, and more.
 
 ```javascript
-const div = document.createElement('div');                     
 // creates a new div referenced in the variable 'div'
+const div = document.createElement("div");
 ```
 
 #### Adding inline style
 
 ```javascript
-div.style.color = 'blue';                                      
-// adds the indicated style rule
+// adds the indicated style rule to the element in the div variable
+div.style.color = "blue";
 
-div.style.cssText = 'color: blue; background: white;';          
 // adds several style rules
+div.style.cssText = "color: blue; background: white;";
 
-div.setAttribute('style', 'color: blue; background: white;');    
 // adds several style rules
+div.setAttribute("style", "color: blue; background: white;");
 ```
 
-See DOM Enlightenment's [section on CSS Style rules](https://domenlightenment.com/#6.2) for more info on inline styles.
-
-Note that if you're accessing a kebab-cased CSS rule from JS, you'll either need to use camelCase or you'll need to use bracket notation instead of dash notation.
+Note that when accessing a kebab-cased CSS rule from JS, you will need to either use camelCase or bracket notation instead of dash notation.
```

Comment:
Just noticed this error (not something you made but definitely needs fixing):
```suggestion
Note that when accessing a kebab-cased CSS rule from JS, you will need to either use camelCase or bracket notation (where you can use either camelCase or kebab-case inside the brackets).
```

Reply:
Actually, shouldn't this be a CSS **property**?

What do you think about the following:


When accessing a kebab-cased CSS property like `background-color` with JS, you will need to either use camelCase with dot notation or bracket notation. When using bracket notation, you can use either camelCase or kebab-case, but the property name must be a string.

```javascript
// dot notation with kebab case: doesn't work as it attempts to subtract color from div.style.background
// equivalent to: div.style.background - color
div.style.background-color;

// dot notation with camelCase: works, accesses the div's background-color style
div.style.backgroundColor;

// bracket notation with kebab-case: also works
div.style["background-color"];

// bracket notation with camelCase: also works
div.style["backgroundColor"];

Reply:
Good call on the `property` keyword. Works for me! If you make the change, I can do a final once-over and merge if all good.